5_train_preference_comparisons.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
This notebook should not be committed at the top-level (belongs in examples/)?

Reply:
Sorry, got added by mistake. I have removed it.